docs/source/notebooks_and_ipython/notebook-example/add_kedro_to_a_notebook.md

```diff
@@ -0,0 +1,668 @@
+```
+
+```python
+features = [
+    "engines",
+    "passenger_capacity",
+    "crew",
+    "d_check_complete",
+    "moon_clearance_complete",
+    "iata_approved",
+    "company_rating",
+    "review_scores_rating",
+]
+```
+
+```python
+X = model_input_table[features]
+y = model_input_table["price"]
+```
+
+```python
+from sklearn.model_selection import train_test_split
+
+X_train, X_test, y_train, y_test = train_test_split(
+    X, y, test_size=test_size, random_state=random_state
+)
+```
+
+The rest of the model evaluation code can now run as before.
+
+```python
+from sklearn.linear_model import LinearRegression
+
+model = LinearRegression()
+model.fit(X_train, y_train)
+model.predict(X_test)
+from sklearn.metrics import r2_score
+
+y_pred = model.predict(X_test)
+r2_score(y_test, y_pred)
+```
+
+### Use a configuration file for all "magic values"
+If we extend the concept of magic numbers to encompass magic values in general, it seems possible that the variable `features` might also be reusable elsewhere. Extracting it from code into the configuration file named `parameters.yml` leads to the following:
+
+```yaml
+# parameters.yml
+
+model_options:
+  test_size: 0.3
+  random_state: 3
+  features:
+    - engines
+    - passenger_capacity
+    - crew
+    - d_check_complete
+    - moon_clearance_complete
+    - iata_approved
+    - company_rating
+    - review_scores_rating
+```
+The `parameters.yml` file is included in the example folder so you can reference the values with notebook code as follows:
+
+```python
+import yaml
+
+with open("parameters.yml", encoding="utf-8") as yaml_file:
+    parameters = yaml.safe_load(yaml_file)
+
+test_size = parameters["model_options"]["test_size"]
+random_state = parameters["model_options"]["random_state"]
+```
+
+```python
+X = model_input_table[parameters["model_options"]["features"]]
+y = model_input_table["price"]
+```
+
+```python
+from sklearn.model_selection import train_test_split
+
+X_train, X_test, y_train, y_test = train_test_split(
+    X, y, test_size=test_size, random_state=random_state
+)
+```
+
+The rest of the model evaluation code can now run as before.
+
+```python
+from sklearn.linear_model import LinearRegression
+
+model = LinearRegression()
+model.fit(X_train, y_train)
+model.predict(X_test)
+from sklearn.metrics import r2_score
+
+y_pred = model.predict(X_test)
+r2_score(y_test, y_pred)
+```
+
+## Use Kedro configuration
+Kedro offers a [configuration loader](/kedro.config.ConfigLoader) to abstract loading values from a yaml file. You can use Kedro configuration loading without a full Kedro project and this approach replaces the need to load the configuration file with `yaml.safe_load`.
+
+### Use Kedro's configuration loader to load "magic values"
+To use Kedro's `OmegaConfigLoader` to load `parameters.yml` the code is as follows:
+
+```python
+from kedro.config import OmegaConfigLoader
+from kedro.framework.project import settings
```

Comment:
Why do you need to import `settings` here?

Reply:
Good point, that's a legacy line of code, and is now removed.